Content.Shared/_RMC14/Holiday/RMCCustomHolidayComponent.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Robust.Shared.Prototypes;
+
+namespace Content.Shared._RMC14.RMCCustomHoliday;
+
+[Prototype("customHoliday")]
+public sealed class CustomHolidayPrototype : IPrototype
+{
+    [IdDataField] public string ID { get; private set; } = default!;
+    [DataField("name")] public string Name { get; private set; } = default!;
+    [DataField("beginDay")] public int BeginDay { get; private set; }
+    [DataField("beginMonth")] public string BeginMonth { get; private set; } = default!;
+    [DataField("description")] public string Description { get; private set; } = default!;
```

Comment:
You don't need to specify the name of the data field if they are the same as the name of the field
```suggestion
    [DataField] public string Name { get; private set; } = default!;
    [DataField] public int BeginDay { get; private set; }
    [DataField] public string BeginMonth { get; private set; } = default!;
    [DataField] public string Description { get; private set; } = default!;
```